connectors/sync_job_runner.py

```diff
@@ -115,7 +115,7 @@ async def execute(self):
             )
             self.data_provider.set_logger(self.sync_job.logger)
             if not await self.data_provider.changed():
-                self.sync_job.log_debug("No change in remote source, skipping...")
+                self.sync_job.info("No change in remote source, skipping sync")
```

Comment:
```suggestion
                self.sync_job.log_info("No change in remote source, skipping sync")
```

Not related to this PR, but seems we never implement `changed` method.

Reply:
I'd even drop the whole `changed()` or try to implement where possible.

I think some SQL connectors and MongoDB can do this - they have metadata about table changes and we can speed things up tremendously there.

Reply:
>I think some SQL connectors and MongoDB can do this 

If there are some connectors which can make use of `changed`, it makes sense to keep it.

Reply:
Hi @artem-shelkovnikov , I think this will fail as there's no `info` method in `SyncJob`

Reply:
Good call, thanks, totally overlooked it! Fixed in https://github.com/elastic/connectors-python/pull/1505